### ISSUE TITLE
"To use relative paths please open a workspace"

### ISSUE THREAD

**Issue by MJCD** [stevensona/shader-toy#25]
I have already saved a new workspace with the glsl file open and saved it, then reopened the workspace - still get this error followed by; 'Unable to open 'GLSL Preview": ENOENT: no such file or directory ...\undefined\other-shader.glsl

**Comment by Malacath-92** [comment]
This issue I introduced in the last release because I did not test the old setup. I made a pull-request to fix this. I would advise you to insert the following snippet into your "settings.json" which you open as described [here](https://code.visualstudio.com/docs/getstarted/settings).
```
    "shader-toy": {
        "useInShaderTextures": true
    },
```
and from now on specify textures in the shader rather than the settings.
